traffic_portal/app/src/modules/public/login/login.tpl.html

```diff
@@ -47,6 +48,13 @@ <h2>Login</h2>
                         </div>
                     </div>
                 </form>
+                <form name="loginForm" role="form" class="form-horizontal form-label-left" novalidate ng-if="oAuthEnabled">
+                    <div class="form-group">
+                        <div class="col-md-6 col-sm-6 col-xs-12 col-md-offset-1">
+                            <button name="loginOauthSubmit" type="submit" class="btn btn-primary" ng-click="loginOauth()">Log in &nbsp;&nbsp;<i class="fa fa-chevron-circle-right"></i></button>
```

Comment:
i think this should be changed to type=button

Reply:
Either that or `ng-click` on the button could be `ng-submit` on the form

Reply:
done